tests/rgw/sanity_rgw.py

```diff
@@ -131,7 +131,8 @@ def run(ceph_cluster, **kw):
     install_common = config.get("install_common", True)
     if install_common:
         if ceph_cluster.rhcs_version.version[0] > 4:
-            setup_cluster_access(ceph_cluster, exec_from)
+            setup_cluster_access(ceph_cluster, rgw_node)
```

Comment:
I have a question+suggestion here for improvement
setup_cluster_access() module perform "yum install -y ceph-common --nogpgcheck" 
https://github.com/red-hat-storage/cephci/blob/master/utility/utils.py#L968 then why do we have explicitly redo the same thing in https://github.com/red-hat-storage/cephci/blob/master/tests/rgw/sanity_rgw.py#L135-L137 @ckulal is there anything around parallel module for which this was added?

@udaysk23 and @ckulal If not can we have it removed?

Reply:
@Ameenasuhani , No this is not included as a part of parallel execution,for parallel support we just moved logic within if condition.